### PR TITLE
Hooks implemented

### PR DESCRIPTION
**Pull request by Ndaboom**
- Refactor the Calculator component (components/Calculator.js) from a class-based component to a functional component with hooks.

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -1,57 +1,53 @@
...
+      <div className="touch-pad">
+        <button onClick={clickHandler} type="button">AC</button>
+        <button onClick={clickHandler} type="button">+/-</button>
+        <button onClick={clickHandler} type="button">%</button>
+        <button onClick={clickHandler} className="orange" type="button">÷</button>
+
+        <button onClick={clickHandler} type="button">7</button>
+        <button onClick={clickHandler} type="button">8</button>
+        <button onClick={clickHandler} type="button">9</button>
+        <button onClick={clickHandler} className="orange" type="button">*</button>
```
Comment (TedLivist) [inline src/components/calculator.js]: - [ ] I get this error when trying to perform a multiplication operation. This should be `x`, not `*`.

  ffofor

File: src/components/calculator.js [hunk 2]
```diff
@@ -1,57 +1,53 @@
...
+
+        <button onClick={clickHandler} type="button">0</button>
+        <button onClick={clickHandler} type="button">.</button>
+        <button onClick={clickHandler} className="orange" type="button">=</button>
       </div>
-    );
-  }
+
+    </div>
+  );
```
Comment (TedLivist) [inline src/components/calculator.js]: - [ ] **[OPTIONAL]** I would recommend you fix your design and stick to the layout that was provided earlier.

  Your design | Required design
  :---:|:---:
  eieoe | rorpr

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Ndaboom,

Good job so far!
✔️ Good documentation
✔️ Refactor to functional component
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
Hi @Ndaboom, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Adressed changes ✔
- No linter errors ✔

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**